src/devices/device_models/AC_branches.jl

```diff
@@ -33,14 +37,14 @@ function flow_variables(ps_m::CanonicalModel,
                         devices::PSY.FlattenIteratorWrapper{B}) where {B <: PSY.ACBranch,
                                                              S <: StandardPTDFForm}
 
-    var_name = Symbol("Pbr_$(B)")
+    var_name = Symbol("Fp_$(B)")
 
     add_variable(ps_m,
                 devices,
                 var_name,
-                false,
-                ub_value = d -> PSY.get_rate(d),
-                lb_value = d -> -1.0*PSY.get_rate(d))
+                false)
+                #ub_value = d -> PSY.get_rate(d), # Add flow bounds in rate constraints
```

Comment:
we should keep the bounds always, PM style. We can make tests to see if they hurt or not.

Reply:
We can't populate bounds here if we want to support `StaticXXXUnbounded` since we aren't dispatching on device formulationf or the variable creation. Instead, we'll need to update the bounds in the flow/rate constraint functions.